packages/styles/variables.css

```diff
@@ -11,7 +11,9 @@
   --gray-20: #f2f2f2;
   --gray-30: #e0e0e0;
   --gray-40: #cccccc;
+  --gray-50: #b3bfc6;
```

Comment:
This doesn't look like a grayscale color, should this just be local to the sidebar?